docs/README.md

```diff
@@ -35,10 +35,11 @@ Using simple YAML configuration, create your own schedule data pipelines (a sequ
 perform), application servers,  and more.
 
 ## Getting Started
-A simple getting stated guide for Liminal can be found [here](getting_started.md)
+A simple hello world guide for Liminal can be found [here](getting-started/hello_world.md) \
+A simple Iris classification model guide for Liminal can be found [here](getting-started/iris_classification.md)
```

Comment:
A more advanced example which demonstrate a simple data-science workflow can be found [here](getting-started/iris_classification.md